src/components/common/ChatCard/ChatCard.tsx

```
@@ -0,0 +1,104 @@
+import { BookmarkCheckIcon, ReloadIcon } from '@/components/common/Icon';
+import { cn } from '@/lib';
+import {
+  chatCardIconColor,
+  chatCardTitleColor,
+  chatCardVariants,
+  CHAT_CARD_STATUS,
+  type ChatCardColor,
+  type ChatCardStatus,
+} from './chatCardVariants';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)